main.java

```diff
@@ -11,33 +13,67 @@ public static void main(String[] args) {
 
         System.out.println("Choose your magical creature to use for battle!");
         System.out.println("-".repeat(93));
-        //list of creatures
-        System.out.println("Troll");
-        System.out.println("Dragon");
-        System.out.println("Hippogriff");
-        System.out.println("Unicorn");
-        System.out.println("Cerberus");
-        System.out.println("Mandrake");
-
-//user input
+         
+     //creating the creatures 
+        var myDragon = new Dragon();
+        var myHippogriff = new Hippogriff();
+        var myMandrake = new Mandrake();
+        var myThreeHeadedDog = new ThreeHeadedDog();
+        var myTroll = new Troll();
+        var myUnicorn = new Unicorn();
+
+
     System.out.println();
-    Scanner myObj = new Scanner(System.in);
+
+    //added scanner to scane user input 
+    Scanner input = new Scanner(System.in);
+
+    //make array of creatures to number
+        ArrayList<Creature> availableCreatures = new ArrayList<Creature>();  
+        availableCreatures.add(myDragon);
+        availableCreatures.add(myHippogriff);
+        availableCreatures.add(myMandrake);
+        availableCreatures.add(myThreeHeadedDog);
+        availableCreatures.add(myTroll);
+        availableCreatures.add(myUnicorn);
+        System.out.println(availableCreatures);
+
     
-    String creatureName1;
+
+    int selectedCreatureIndex;
+   
     // Enter first creature and press Enter
     System.out.println("Enter first creature: "); 
-    creatureName1 = myObj.nextLine();   
-       
-    System.out.println("Chosen creature 1: " + creatureName1); 
-       System.out.println();
-    
-  
-    String creatureName2;
-    // Enter second creautre and press Enter
+    selectedCreatureIndex = input.nextInt();
+    selectedCreature = availableCreatures[selectedCreatureIndex];
```

Comment:
Since you're using an `ArrayList` rather than a simple array, you'll have look at the reference for the `ArrayList` class: https://www.w3schools.com/java/java_arraylist.asp
```suggestion
    selectedCreature = availableCreatures.get(selectedCreatureIndex);
```